bin/check-redis-list-length.rb

```diff
@@ -63,19 +63,38 @@ class RedisListLengthCheck < Sensu::Plugin::Check::CLI
          description: 'Redis list KEY to check',
          required: true
 
+  option :threshold,
```

Comment:
It they are the same we can either throw an error or assume have a default (let's say above because it's the current behaviour).